task1.js

```diff
@@ -0,0 +1,73 @@
+//  Write a JavaScript program to pick a specified number of unique random elements from a given array (so that no element is selected more than once).
+//  Sample Output:
+//  Original array:
+//  [12, 34, 23, 56]
+//  2 random elements from the array.
+//  [34, 12]
+//  3 random elements from the array.
+//  [56, 12, 34]
+
+//  If an element is already selected, it cannot be selected again.
+
+const findRandomElements = (arr, elemCount) => {
```

Comment:
i would suggest you to use `switch/case `for construction that have a lot of `if`s`